exercises/practice/reverse-string/.meta/template-data.yaml

```diff
@@ -0,0 +1,27 @@
+properties:
+  reverse:
+    test: |-
+      sprintf(q:to/END/, %case<description>.raku, %case<input><value>.raku, %case<expected>.raku, %case<input><value>.raku, %case<input><value>.raku);
+      subtest %s => {
+        cmp-ok(
+            switch-round(%s),
+            "eq",
+            %s,
+        );
+        cmp-ok(
+            switch-round(switch-round(%s)),
+            "eq",
+            %s,
+        );
+      }
+      END
+
+unit: module
+example: |-
+  sub switch-round($string) is export {
+      $string.flip
+  }
+
+stub: |-
+  sub switch-round($string) is export {
```

Comment:
Perhaps we could call the sub `reverse-string`? I don't think it overlaps with anything that exists and fits with the exercise name better than `switch-round` IMO.